packages/angular/projects/vcd/sdk/src/new/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Is this needed here?

Reply:
Yes it is, because the plugin builder for angular 7+ has different structure and dependencies